netbout-client/src/main/java/com/netbout/client/RtAlias.java

```diff
@@ -108,11 +110,24 @@ public void photo(final URI uri) {
 
     @Override
     public String email() throws IOException {
-        return this.request.fetch()
-            .as(XmlResponse.class)
-            .xml()
-            .xpath("/page/alias/email/text()")
-            .get(0);
+        final XML body = this.request.fetch().as(XmlResponse.class).xml();
+        final List<String> lverified = body
```

Comment:
@CarlosGines Extracting `alias.email()` because it is used 3 times and makes the code less readable when chaining the calls. I suggested inlining `body.xpath` to avoid using bad variable names as `lverified` and `lunverified`. If it is too much overhead, let's rename variables to `verified`, `unverified` for the nodes and `old`, `mail` for the strings.